### ISSUE TITLE
Crash durring install

### ISSUE THREAD

**Issue by Merhern** [rockerbacon/modorganizer2-linux-installer#465]
Program says I should look into terminal output, but what am I supposed to see here? Is it gtk problem?

```
INFO: extracting '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05.7z' to '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05'
INFO: removing existing JDK installation in '/home/martin/.local/share/Steam/steamapps/compatdata/489830/pfx/drive_c/java'
INFO: installing JDK in '/home/martin/.local/share/Steam/steamapps/compatdata/489830/pfx/drive_c/java'
INFO: installing Mod Organizer 2 in '/home/martin/Games/mod-organizer-2-skyrimspecialedition/modorganizer2'
INFO: installing script extender in '/home/martin/.local/share/Steam/steamapps/common/Skyrim Special Edition'
INFO: copying '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05/skse64_2_01_05/Data' into '/home/martin/.local/share/Steam/steamapps/common/Skyrim Special Edition'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05.7z'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/Mod.Organizer-2.4.4.7z'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/Mod.Organizer-2.4.4'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/OpenJDK8U-jre_x64_windows_hotspot_8u312b07.zip'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/OpenJDK8U-jre_x64_windows_hotspot_8u312b07'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/winetricks'

(zenity:3966): Gtk-WARNING **: 22:31:32.047: Theme directory apps/24 of theme breeze-extended-nord has no size field


(zenity:3966): Gtk-WARNING **: 22:31:32.051: Theme directory apps/24@2x of theme breeze-extended-nord has no size field


(zenity:3966): Gtk-WARNING **: 22:31:32.051: Theme directory apps/24@3x of theme breeze-extended-nord has no size field
```

**Comment by UnstoppableMango** [comment]
Based on this line here, I think you're having the same problem I was

`INFO: copying '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05/skse64_2_01_05/Data' into '/home/martin/.local/share/Steam/steamapps/common/Skyrim Special Edition'`

TLDR; The SKSE copy step is failing for whatever reason. In my case the files already existed, so I removed `Data/Scripts` and the `skse*` files from `<Skyrim SE game dir>`

```shell
rm -r "$YOUR_STEAM_DIR/steamapps/common/Skyrim Special Edition/Data/Scripts"
rm "$YOUR_STEAM_DIR/steamapps/common/Skyrim Special Edition/skse*"
```

I had done a prior install, but decided I wanted to move directories around and tried to re-install. I noticed the "removing *" logs were coming from a cleanup step, and the last actual log was that copy line. Found that log in `step/install_external_resources.sh` and saw it was attempting to loop over `game_scriptextender_files`. I found _that_ variable in `gamesinfo/skyrimspecialedition.sh` with 4 values in the array; the `Data` dir and 3 skse binaries. I noticed those already existed in my install, so I tried to run `cp -an` manually. `cp` exited with code 1, and the install therefore exited. I removed the files, `cp` no longer exits with code 1, and the install finishes.

I'm not great at bash scripting, but I imagine a fix would be something to do with `set +e` so the script doesn't immediately exit when `cp` fails, and log some generic error about the copy failing. It at least would point people in the right direction when the prompt says to "look in the console"

**Comment by BasedRatGod** [comment]
For me the issue was because I wasn't using the original library for my Steam games, I have two SSDs, a smaller one I use for speed, and a bigger one for storage. On the smaller one, Steam is installed, and the bigger one is where I install my games.
As it turns out, the install script only sees the first, original library, and not the library I was actually using, which obviously causes issue because the game isn't actually installed there, so I had to manually set the Steam library path in /step/load_gameinfo.sh
Specifically, I had to change these two lines for it to work:
`game_prefix="$steam_library/steamapps/compatdata/$game_appid/pfx"
game_installation="$steam_library/steamapps/common/$game_steam_subdirectory"`
